.buildkite/test-check/scripts/test-check-xpu.sh

```
@@ -0,0 +1,31 @@
+
+echo "Setup test environment..."
+docker exec $CONTAINER_NAME \
+              bash -c "uv pip install --upgrade pip setuptools && \
+              uv pip install .[dev] --extra-index-url https://download.pytorch.org/whl/xpu"
+
+echo "Run tests inside the container..."
+docker exec -e NUMA_NODE=${NUMA_NODE} -e NUMA_CPUSET=${NUMA_CPUSET} -e ZE_AFFINITY_MASK=${ZE_AFFINITY_MASK} $CONTAINER_NAME \
+              bash -c "numactl --physcpubind=${NUMA_CPUSET:-84-111} --membind=${NUMA_NODE:-3} make test-xpu"
+
+echo "Clean up the container after tests..."
+docker rm -f $CONTAINER_NAME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)